docs/content/users/usage/commands.md

```diff
@@ -233,13 +233,13 @@ ddev config global --omit-containers=dba,ddev-ssh-agent
 * `--letsencrypt-email`: Email associated with Let’s Encrypt; `ddev global --letsencrypt-email=me@example.com`.
 * `--mutagen-enabled`: If `true`, web container will use Mutagen caching/asynchronous updates.
 * `--nfs-mount-enabled`: Enable NFS mounting on all projects globally.
-* `--no-bind-mounts`: If `true`, don’t use bind-mounts. Useful for environments like remote Docker where bind-mounts are impossible. (default `true`)
+* `--no-bind-mounts`: If `true`, don’t use bind-mounts. Useful for environments like remote Docker where bind-mounts are impossible. (default `false`)
```

Comment:
Got it. Maybe the docs can explicitly specify that `--no-bind-mounts` is the same as `--no-bind-mounts=true`, because I thought about the default in the global configuration.

Reply:
Yeah, all? of the `ddev config` flags that are binary default to true I think.